install/kubernetes/helm/istio/charts/kiali/templates/clusterrolebinding.yaml

```diff
@@ -14,6 +14,6 @@ roleRef:
   name: kiali
 subjects:
 - kind: ServiceAccount
-  name: kiali-service-account
+  name: kiali
```

Comment:
Why change the service account name?

Reply:
All kiali resources have the name "kiali" including the service account (see: https://github.com/kiali/kiali/blob/master/deploy/kubernetes/kiali.yaml#L4) -- notice in this helm chart file, all the names are "kiali" except this one (the service account)... to remain consistent, we'll want everything named kiali.

Reply:
I prefer we keep consistent in the istio chart, for other chart, we are using `xxx-service-account` for other charts, such as pilot, mixer etc